Fix sonarqube Code smells issues

KiruthikaJeyashankar opened the pull request:
As a apart of this PR, critical severity & some of the major severity code smells (Maintainability) are fixed

Report: https://sonarcloud.io/summary/overall?id=mosip_mimoto
Issue link : https://mosip.atlassian.net/browse/INJI-219

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] fix major severity code smells as per SonarQube rules

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] add test for MimotoConfigTest

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] fix major severity code smells as per SonarQube rules

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] remove commented code

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] remove unused private method

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] fix null pointer exception reliability issue

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] replace private constructor in utility classes with Lombok annotation

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] move constant field to variable in method to avoid global scope in class

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] move setting up mock to setup method in test

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] remove unused empty constructor

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] rename method

vijay151096 reviewed (approved):
looks good to me, please make sure, all the unit test cases are passing successfully.

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] replace check null method with libraries isEmpty check

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] extract method and constants

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] move logger related constants to logger constants file

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] revert add test for MimotoConfigTest

KiruthikaJeyashankar committed:
refactor(INJI-219): [Kiruthika] rename MimotoConfigTest to MimotoConfig

MimotoConfigTest did not have any tests written & has only beans declared hence making this change
